README.md

```diff
@@ -6,6 +6,7 @@
 
 - Ruby 2.4.2+
 - Middleman 3.4.1
+- Contentful API keys setup
```

Comment:
It is step 3 in `installation`, is this redundant?  I would suggest tweaking that if it's unclear—or just moving that to here (I would agree with moving it as I think it's much more important than where it is now).

```md
- Setup local environment keys.  See [howtos/environment.md](https://github.com/sharesight/www.sharesight.com/blob/master/howtos/environment.md).
```

Reply:
Oops - I just missed that. I actually think its OK where it is, and the `environment.rb` file is very clear